### PR TITLE
Package reg

### PR DESCRIPTION
**Pull request by jpe442**
## Description
https://jira.mesosphere.com/browse/DCOS-40917

## Urgency
- [ X] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

File: pages/1.12/administering-clusters/package-registry/quickstart/index.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+echo '{"registry":{"service-account-secret-path":"registry-private-key"}}' > registry-options.json
+```
+
+#### 2. Install Package Registry
+
+```bash
+dcos package install package-registry --options=registry-options.json --yes
+```
+
+Note: The default configuration for the DC/OS Package Registry stores DC/OS Packages on the local filesystem. This configuration doesn't scale horizontally nor is it highly available. Please see S3 Storage Option section.
```
Comment (takirala) [inline pages/1.12/administering-clusters/package-registry/quickstart/index.md]: should `S3 storage option` be hyperlinked ?

File: pages/1.12/administering-clusters/package-registry/quickstart/index.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+#### 5. Give full permission to the service account
+
+```bash
+dcos security org users grant registry-account dcos:adminrouter:ops:ca:rw full
+```
+
+NOTE: The secret information associated with the service account is stored in a path called "registry-private-key" in the DC/OS Secret store. Please change "registry-private-key" to your prefered path.
+
+These instructions create two sensitive files on the local file system: "private-key.pem" and "public-key.pem". Please make sure to save these files in a secure place or delete them. They are not needed after being stored in the DC/OS Secret Store.
```
Comment (takirala) [inline pages/1.12/administering-clusters/package-registry/quickstart/index.md]: (nit) can we make "private-key.pem" to be `private-key.pem` instead? i.e., with the code formatting. (not sure how we do this in other parts of docs.

Same comment applicable for following in this document.
- `public-key.pem`
- `registry`
- `registry-private-key`

File: pages/1.12/administering-clusters/package-registry/using/index.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+Installing the DC/OS Package Registry also installs the CLI for it (`dcos registry`). The CLI can be installed again with:
+
+```bash
+dcos package install package-registry --cli
+```
+
+### Downloading Packages
+
+For a list of DC/OS Packages supported by the DC/OS Package Registry see here. To download a version of Jenkins:
```
Comment (takirala) [inline pages/1.12/administering-clusters/package-registry/using/index.md]: `here` is not a valid hyperlink. Should refer to https://downloads.mesosphere.com/universe/packages/packages.html

File: pages/1.12/administering-clusters/package-registry/configure-and-deploy/index.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+layout: layout.pug
+navigationTitle: Configuring and Deploying Package Registry
+title: Configuring and Deploying Package Registry
+menuWeight: 30
+excerpt: Configuration Options for DC/OS Package Registry
+beta: true
+enterprise: true
+---
+
+For a full list of the configuration options available for the DC/OS Package Registry use the `dcos package describe --config`. For a detail description of how to configure and deploy DC/OS Servers see [Configure Universe Services](https://docs.mesosphere.com/1.10/deploying-services/config-universe-service/).
```
Comment (takirala) [inline pages/1.12/administering-clusters/package-registry/configure-and-deploy/index.md]: `Services` instead of `Servers` ?

Comment (jpe442) [reply pages/1.12/administering-clusters/package-registry/configure-and-deploy/index.md]: yep

File: pages/1.12/administering-clusters/package-registry/quickstart/index.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+```bash
+dcos package repo remove Universe
+```
+
+# Installing DC/OS Package Registry
+
+### Enable the Read-Only Bootstrap Package Registry
+
+DC/OS Enterprise is pre-configured to run a read-only Package Registry which contains two DC/OS Packages. The DC/OS Enterprise CLI and the DC/OS Package Registry. To enable this repository with the DC/OS Package Manager, we must add it to the list to repository.
```
[the remark this answers is not in the document]
Comment (jpe442) [reply pages/1.12/administering-clusters/package-registry/quickstart/index.md]: agreed on the use of "Package Registry" throughout. What would be a good list of repos to pull from?

File: pages/1.12/administering-clusters/package-registry/quickstart/index.md (hunk 1 above)
Comment (jpe442) [reply pages/1.12/administering-clusters/package-registry/quickstart/index.md]: done. agreed

File: pages/1.12/administering-clusters/package-registry/quickstart/index.md (hunk 2 above)
Comment (jpe442) [reply pages/1.12/administering-clusters/package-registry/quickstart/index.md]: yep... I fixed a bunch the first time through, but am going to go through the doc and try to find all of them.

File: pages/1.12/administering-clusters/package-registry/using/index.md (hunk 3 above)
Comment (jpe442) [reply pages/1.12/administering-clusters/package-registry/using/index.md]: got it

File: pages/1.12/administering-clusters/package-registry/quickstart/index.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (takirala) [reply pages/1.12/administering-clusters/package-registry/quickstart/index.md]: Just to be clear, my comment here was to rephrase the sentence from `...... list of repository` to `........ list of repositories`

> What would be a good list of repos to pull from?

Sorry, I don't understand what you mean by above ?!

[the remark this answers is not in the document]
Comment (jpe442) [reply pages/1.12/administering-clusters/package-registry/quickstart/index.md]: oh... I completely misunderstood that! thanks for clarification. I can easily fix that.

[the remark this answers is not in the document]
Comment (jpe442) [reply pages/1.12/administering-clusters/package-registry/quickstart/index.md]: done

**Pull request merged**